configs/config/pretrain/byol/byol_1gpu.yaml

```diff
@@ -0,0 +1,121 @@
+# @package _global_
+config:
+  MAX_ITER: 312785 # int(1000. * Training set length / 4096) , here set according to ImageNet ILSVRC-2012 dataset
+  WARMUP_ITERS: 3128 # int(10. * Training set length / 4096) , here set according to ImageNet ILSVRC-2012 dataset
+  VERBOSE: False
+  LOG_FREQUENCY: 10
+  TEST_ONLY: False
+  TEST_MODEL: False
+  SEED_VALUE: 1337
+  MULTI_PROCESSING_METHOD: forkserver
+  HOOKS:
+    PERF_STATS:
+      MONITOR_PERF_STATS: True
+      ROLLING_BTIME_FREQ: 313
+    TENSORBOARD_SETUP:
+        USE_TENSORBOARD: True
+        EXPERIMENT_LOG_DIR: "byol_reference"
+        LOG_PARAMS: False
+        FLUSH_EVERY_N_MIN: 20
+  DATA:
+    NUM_DATALOADER_WORKERS: 5
+    TRAIN:
+      DATA_SOURCES: [disk_folder]
+      DATASET_NAMES: [imagenet1k_folder]
+      BATCHSIZE_PER_REPLICA: 64
+      LABEL_TYPE: sample_index    # just an implementation detail. Label isn't used
+      TRANSFORMS:
+        - name: ImgReplicatePil
+          num_times: 2
+        - name: RandomResizedCrop
+          size: 224
+        - name: RandomHorizontalFlip
+          p: 0.5
+        - name: ImgPilColorDistortion
```

Comment:
With the defaults in `img_pil_color_distortion.py`, we have saturation=0.4, when it should be 0.2 (pg 17. Table 6).

We need to 

1. Parameterize img_pil_color_distortion.py to take in brightness, contrast, saturation, and hue (the same way we are passing in the strength argument), and then set saturation: 0.4.

Reply:
nit: While we are at it, let's also make `p_color_jitter` and `p_rnd_gray` configurable as well.